docs/dev/docker.rst

```diff
@@ -0,0 +1,45 @@
+Docker Setup for local development
+==================================
+The following describes how to set up an instance of the site on your
+computer for development using the provided docker and docker-compose 
+files.
+
+Prerequisites
+-------------
+This guide assumes you have already installed and set up the following:
+
+1. Git_
+2. Docker_
+3. Docker-Compose_
+
+These docs assume a Unix-like operating system, although the site should, in
+theory, run on Windows as well. All the example commands given below are
+intended to be run in a terminal.
+
+Installation
+------------
+1. Clone this repository or your fork_:
+
+   .. code-block:: bash
+
+      git clone https://github.com/mozilla/normandy.git
+      cd normandy
+
+2. Start the docker images via ``docker-compose``:
+
+   .. code-block:: bash
+
+    docker-compose -f docker-compose.yml -f docker-compose.development up -d --build
```

Comment:
Why specify `-f` twice?

Reply:
I'd rather have someone use this command so they also know they are overriding the non ```.development``` docker-compose file.

Reply:
So saying `-f` twice overlays the two configs? That isn't clear from your docs or from `docker-compose`'s docs. Can you add a note about it here?

Reply:
FYI, I'm about to remove `ci/docker-compose.yml`, since I just refactored all of our CI to not use docker-compose. It probably makes sense to merge this two compose files.

Reply:
The files here refer to the docker-compose files as if they are in the current directory. However, they are in the `ci` directory, which isn't mentioned anywhere in these docs.

It also seems weird that we are using CI resources for local development.